falco/templates/daemonset.yaml

```diff
@@ -60,7 +60,14 @@ spec:
           resources:
 {{ toYaml .Values.resources | indent 12 }}
           securityContext:
-            privileged: true
+            privileged: {{ not .Values.leastPrivileged.enabled }}
+            {{- if .Values.leastPrivileged.enabled }}
+            capabilities:
+              add:
+              - BPF
```

Comment:
Note that this requires runc 1.0.0-rc93 onwards: https://github.com/opencontainers/runc/pull/2553/files.  
In older versions, this won't work.